mne/event.py

```diff
@@ -319,6 +319,8 @@ def write_events(filename, event_list):
 
 
 def _find_stim_steps(data, first_samp, pad_start=None, pad_stop=None, merge=0):
+    data = np.append(np.array([0])[None, ...], data, axis=1)
+    first_samp -= 1
```

Comment:
woops, try not to touch this part. `events.py` is well tested, I doubt there is a bug here. You have many failing tests because of this ...